### PR TITLE
Don't modify the main query if is in a single shortcode.

### PR DESCRIPTION
**Pull request by mitogh**
🎟  https://central.tri.be/issues/91787

### CODE REVIEW

**Comment by tr1b0t** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | There must be a single space after the first parenthesis | [Query.php:77](https://github.com/moderntribe/the-events-calendar/blob/e8b92c4b03ced280f3c7ca5dcb37166eacd01a69/src/Tribe/Query.php#L77)
:x: | There must be a single space before the last parenthesis | [Query.php:77](https://github.com/moderntribe/the-events-calendar/blob/e8b92c4b03ced280f3c7ca5dcb37166eacd01a69/src/Tribe/Query.php#L77)

> via jenkins/codesniffer-phpcs

File: readme.txt [hunk 1]
```diff
@@ -310,6 +310,8 @@ The plugin is made with love by [Modern Tribe Inc](http://m.tri.be/2s).
 
 = [4.6.11] TBD =
 
+* Fix - Don't modify the main query if is in a single post to allow custom variables don't affect the page loading when using shortcode [91787]
```
Comment (barryhughes) [inline readme.txt]: Let's acknowledge one or more customers whenever we can :-)

Comment (barryhughes) [reply readme.txt]: Joseph Reckley is one, so something like _"(our thanks to Joseph Reckley and others for highlighting this problem)"_ would work.

**Comment by bordoni** [comment]
Avoid pushing the `common` submodule hash update.

**Comment by mitogh** [comment]
Closing this one to remove `common` module references.

**Pull request closed without merging**